README.md

```diff
@@ -28,4 +28,5 @@
 | 12/10 | PHPSP + Pub | Presencial | [Meetup PHPSP](https://www.meetup.com/pt-BR/php-sp/events/) | São Paulo - SP |
 | 13/10 | Boteco PHP | Presencial | [Meetup PHP Rio](https://www.meetup.com/php-rio/events/296590312/) | Rio de Janeiro - RJ |
 | 09/11 | PHPSP + Pub | Presencial | [Meetup PHPSP](https://www.meetup.com/pt-BR/php-sp/events/) | São Paulo - SP |
+| 07/12 a 09/12 | PHP Conference Brasil | Online & Presencial | [PHP Conference Brasil](https://phpconference.com.br/) | Pedra Branca - SC |
```

Comment:
A exemplo do PHPeste, acho interessante, para seguir o padrão, manter somente a data de início do evento.
E a exemplo do PHP Community Summit, pode ser interessante ter "Híbrido" ao invés de "Online & Presencial".
O que acha?

```suggestion
| 07/12 | PHP Conference Brasil | Híbrido | [PHP Conference Brasil](https://phpconference.com.br/) | Pedra Branca - SC |
```

Reply:
Fechado. Mantendo o padrão.